language/move-vm/runtime/src/unit_tests/vm_arguments_tests.rs

```diff
@@ -474,3 +479,64 @@ fn check_constant_args() {
         StatusCode::TYPE_MISMATCH,
     );
 }
+
+#[test]
+fn check_signer_args() {
+    let two_signer_script = make_script(Signature(vec![
+        SignatureToken::Reference(Box::new(SignatureToken::Signer)),
+        SignatureToken::Reference(Box::new(SignatureToken::Signer)),
+    ]));
+    // too few signers (0)
+    assert_eq!(
+        call_script_(two_signer_script.clone(), vec![], vec![], vec![])
+            .err()
+            .unwrap()
+            .major_status(),
+        StatusCode::TYPE_MISMATCH
+    );
+
+    // too few signers (1)
+    let one_signer = vec![AccountAddress::random()];
```

Comment:
It probably doesn't matter here, but non-determinism in tests makes me panic

Reply:
I was copying the style of the pre-existing tests here, but agreed that a hardcoded address would probably make more sense.